packages/daemon/src/context/InvitesHandler.js

```diff
@@ -475,7 +481,14 @@ export default class InvitesHandler {
     }
 
     const addr = inviteRequest.senderAddress.substr(2)
-    const messageHex = hash(Buffer.from(addr, 'hex'))
+    const addressHash = bufferFromHex(hash(Buffer.from(addr, 'hex')))
```

Comment:
Not a big issue so not need to make any change now, just a general comment: it seems the `hash()` function returning an hex string is not the best here, as it's doing a buffer -> hex string -> buffer conversion.
Might be easier to have a `hashBuffer()` function for example returning the buffer, and `hash()` using `hashBuffer()` and returning the hex string.
The [`@erebos/hex` package](https://erebos.js.org/docs/hex) can be used to simplify this kind of conversion logic.